.github/workflows/full-build.yml

```diff
@@ -380,3 +392,21 @@ jobs:
         if: always()
         run: type gaphor-self-test.txt
         shell: cmd
+
+  publish-to-pypi:
+    name: Publish to PyPI (release only)
+    needs: [ linux-gtk4, linux-flatpak-devel, macos-gtk4, check-linux-appimage, check-macos-app, check-windows-installer ]
```

Comment:
Since the 3 check-* jobs are the last builds to run in the pipeline, I don't think you need the other jobs here

Reply:
The check-* jobs do not depend on the other jobs. I figured it's best if they also pass before uploading to PyPI, just t be on the safe side. Strictly speaking we only need the check-* jobs.